dev-tools/action-install-fluvio-cluster.sh

```diff
@@ -16,6 +16,12 @@ if [ "$DEVELOPMENT" = "true" ]; then
                 # Install Fluvio Src
                 cargo install --path ./src/cli
                 popd
+        elif [ "$VERSION" = "v0.6.0-alpha.3" ]; then 
+                pushd /tmp/
+                wget https://raw.githubusercontent.com/infinyon/fluvio/feature/alpha_install_script/dev-tools/install-fluvio.sh
```

Comment:
will update this to `master` branch once https://github.com/infinyon/fluvio-demo-apps-node/pull/11 is working.